coderd/database/queries/organizations.sql

```diff
@@ -66,6 +66,14 @@ WHERE
             user_id = $1
     );
 
+-- name: GetOrganizationResourceCountByID :one
+SELECT
+    (SELECT COUNT(*) FROM workspaces WHERE workspaces.organization_id = $1 AND workspaces.deleted = false) AS workspace_count,
+    (SELECT COUNT(*) FROM groups WHERE groups.organization_id = $1) AS group_count,
+    (SELECT COUNT(*) FROM templates WHERE templates.organization_id = $1 AND templates.deleted = false) AS template_count,
+    (SELECT COUNT(*) FROM organization_members WHERE organization_members.organization_id = $1) AS member_count,
+    (SELECT COUNT(*) FROM provisioner_keys WHERE provisioner_keys.organization_id = $1) AS provisioner_key_count;
+
```

Comment:
I wonder if we should declare this as a function so that it could be used in `protect_deleting_organizations`. this is super nice, readable, and concise, but the code for accomplishing the same counting in `protect_deleting_organizations` is like 20 lines longer. 🤔